rules/Naming/ExpectedNameResolver/InflectorSingularResolver.php

```diff
@@ -37,11 +36,6 @@ final class InflectorSingularResolver
      */
     private const CAMELCASE = 'camelcase';
 
-    public function __construct(
-        private readonly Inflector $inflector
-    ) {
-    }
-
     public function resolve(string $currentName): string
```

Comment:
As new logic is added here, we'll need a test case to cover at lest few basic case.